scripts/nn/layers/l2_loss.dml

```diff
@@ -70,3 +70,21 @@ backward = function(matrix[double] pred, matrix[double] y)
   dpred = (pred-y) / N
 }
 
+compute_loss_accuracy = function(matrix[double] probs, matrix[double] y)
```

Comment:
what is this for?  if it's is supposed to return the loss and the accuracy, let's just have Caffe2DML call the appropriate loss function and separately an accuracy function.  I don't think it's a good idea to pollute the `nn` library to account for Caffe issues.

Reply:
The reason I added a separate loss and accuracy function is to allow DML writers flexibility to modify them for their problem. This goes in direction of making Caffe2DML a thin wrapper :)